docs/examples/viz_dt_ellipsoids.py

```diff
@@ -0,0 +1,227 @@
+# Images.
+
+fetch_viz_dmri()
+
+###############################################################################
+# The tensor ellipsoids are expressed as eigenvalues and eigenvectors which are
+# the decomposition of the diffusion tensor that describes the water diffusion
+# within a voxel.
+
+slice_evecs, _ = load_nifti(read_viz_dmri('slice_evecs.nii.gz'))
+slice_evals, _ = load_nifti(read_viz_dmri('slice_evals.nii.gz'))
+roi_evecs, _ = load_nifti(read_viz_dmri('roi_evecs.nii.gz'))
+roi_evals, _ = load_nifti(read_viz_dmri('roi_evals.nii.gz'))
+whole_brain_evecs, _ = load_nifti(read_viz_dmri('whole_brain_evecs.nii.gz'))
+whole_brain_evals, _ = load_nifti(read_viz_dmri('whole_brain_evals.nii.gz'))
+
+###############################################################################
+# Using tensor_slicer actor
+# =========================
+# First we must define the 3 parameters needed to use the ``tensor_slicer``
+# actor, which correspond to the eigenvalues, the eigenvectors, and the sphere.
+# For the sphere we use ``prim_sphere`` which provide vertices and triangles of
+# the spheres. These are labeled as 'repulsionN' with N been the number of
+# vertices that made up the sphere, which have a standard number of 100, 200,
+# and 724 vertices.
+
+vertices, faces = prim_sphere('repulsion100', True)
+
+
+###############################################################################
+# As we need to provide a sphere object we create a class Sphere to which we
+# assign the values obtained from vertices and faces.
+
+class Sphere:
+    vertices = None
+    faces = None
+
+
+sphere = Sphere()
+sphere.vertices = vertices
+sphere.faces = faces
+
+###############################################################################
+# Now we are ready to create the ``tensor_slicer`` actor with the values of a
+# brain slice. We also define the scale so that the tensors are not so large
+# and overlap each other.
+
+tensor_slice = actor.tensor_slicer(evals=slice_evals, evecs=slice_evecs,
+                                   sphere=sphere, scale=.3)
+
+###############################################################################
+# Next, we set up a new scene to add and visualize the tensor ellipsoids
+# created.
+
+scene = window.Scene()
+scene.add(tensor_slice)
+
+# Enables/disables interactive visualization
+interactive = True
+
+if interactive:
+    window.show(scene)
+
+window.record(scene, size=(600, 600), out_path='tensor_slice_100.png')
+window.record(scene)
+
+###############################################################################
+# To render the same tensor slice using a different sphere we redefine the
+# vertices and faces of the sphere using prim_sphere with other sphere
+# specification, let's say 'repulsion200' and 'repulsion724'.
+#
+# If we zoom in at the scene to see with detail the tensor ellipsoids displayed
+# with the different spheres, we get the following results.
+#
+
```

Comment:
I would like to display an image like this

![image](https://github.com/fury-gl/fury/assets/31288525/08f90598-ec1d-43e6-91b4-502a63be0307)

Is it possible just to upload this image or do I have to create the scene in that way? Also, how can I zoom in the scene to get an image like this?

![image](https://github.com/fury-gl/fury/assets/31288525/b8644a1c-f5c5-4c6a-9571-0d3e17e434d8)

I tried `scene.zoom(2)` but it is not working, what is the correct way to do it?

Reply:
I agree, please, use `ShowManager`

Reply:
put the background white to increase contrast also